src/applications/static-pages/translation.js

```diff
@@ -24,6 +24,14 @@ const configureTranslationLink = (e, targetLang) => {
     });
   };
 };
+// this can be applied site wide and moved to an accessibility helper
+const hidePresentationalArrowsFromScreenReader = () => {
+  const arrows = document.getElementsByClassName('fa-arrow-down');
+  if (!arrows.length) return;
+  for (const arrow of arrows) {
+    arrow.setAttribute('aria-hidden', true);
+  }
+};
```

Comment:
Instead of using JS to correct this after the page loads, I think a more straightforward fix is to just fix that HTML at the time it is generated. The issue is describing the page's "table of contents." It's kind of sneaky in how that is generated. But the HTML for those icons are here -> https://github.com/department-of-veterans-affairs/vets-website/blob/0fe779514bcb1c09a769df0f6267e1ff32287299/src/site/stages/build/plugins/modify-dom/add-id-to-subheadings.js#L94

I think we can just add `aria-hidden="true" ` into that HTML snippet

Reply:
^ agreed

Reply:
resolved... and yes definitely agreed